src/mercury_engine_data_structures/formats/bmtun.py

```diff
@@ -1,12 +1,13 @@
 from __future__ import annotations
 
 import functools
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any
 
```

Comment:
instead of `Any` we can be more specific. you can do something like `Tunable = str | float | bool | int | list[str]`